src/powerquery-parser/language/type/typeUtils/isCompatible.ts

```diff
@@ -79,14 +79,18 @@ export function isCompatibleWithFunctionSignature(
 }
 
 export function isCompatibleWithFunctionParameter(
-    left: Type.FunctionParameter,
+    left: Type.PqType | undefined,
```

Comment:
I realize PqType was already exported, but do we use abbreviations like `Pq` anywhere else? Would the full PowerQueryType make more sense?

Reply:
Talked offline. I'll create another PR for the rename.